npm/angular/src/mount.ts

```diff
@@ -8,7 +8,7 @@ window.Mocha['__zone_patch__'] = false
 import 'zone.js/testing'
```

Comment:
Consistency is crucial; let's ensure this aligns with our established practices.

Reply:
This is a troll bot, ignore 🙄